get-all-ops.js

```diff
@@ -1,39 +1,66 @@
 const fs = require("fs");
 const sax = require("sax");
-const saxparser = sax.parser();
+const saxparser = sax.parser(true);
```

Comment:
Was there a major reason to turn on strict mode? Given how many typos and oddities there are in Intel's intrinsics docs, I don't trust them to produce strict XML or to always use consistent casing on tag names either.

Reply:
Actually, that was my assumption, the correctness of the XML. However, it is also true the possibility of inconsistencies on tags. I agree is better to use the loose mode.